partcad/src/partcad/shape_config.py

```
@@ -10,17 +10,24 @@
+
 
 class ShapeConfiguration:
+    is_manufacturable: bool = False
+
     def __init__(self, config):
         self.config = config
+
         if "name" in config:
             self.name = config["name"]
         else:
             name = "part" + "".join(random.choices(string.ascii_uppercase + string.digits, k=8))
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)